### PR TITLE
Updates for esm chunks

### PR DESCRIPTION
**Pull request by alexeagle**
Don't name folders with dots in the name - users expect that these are files
"chunks" is more intuitive than "cs"

allow native ESmodule output
expose a label to grab the ES2015 chunks

These are needed by the photon POC

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "bazel.buildifierExecutable": "${workspaceFolder}/node_modules/.bin/buildifier",
+    "bazel.buildifierFixOnFormat": true
```
Comment (gregmagolan) [inline .vscode/settings.json]: cool

File: internal/e2e/rollup_code_splitting/additional_entry.spec.js [hunk 2]
```diff
@@ -8,38 +8,46 @@ describe('bundling additional entry point', () => {
     check(path, 'bundle.min.js', 'bundle-min_golden.js_');
   });
 
-  it('bundle.cs.es6 should work', () => {
+  // Disabled because native ESModules can't be loaded in current nodejs
```
Comment (gregmagolan) [inline internal/e2e/rollup_code_splitting/additional_entry.spec.js]: TODO for a future version of nodejs?

File: internal/e2e/rollup_code_splitting/additional_entry.spec.js [hunk 3]
```diff
@@ -8,38 +8,46 @@ describe('bundling additional entry point', () => {
...
+  xit('bundle_chunks_es6 should work', () => {
     const additional_entry = require(
-        'build_bazel_rules_nodejs/internal/e2e/rollup_code_splitting/bundle.cs.es6/additional_entry.js')
+        'build_bazel_rules_nodejs/internal/e2e/rollup_code_splitting/bundle_chunks_es6/additional_entry.js');
+    console.error(additional_entry)
```
Comment (gregmagolan) [inline internal/e2e/rollup_code_splitting/additional_entry.spec.js]: debug log?

File: internal/e2e/rollup_code_splitting/additional_entry.spec.js (hunk 2 above)
Comment (alexeagle) [reply internal/e2e/rollup_code_splitting/additional_entry.spec.js]: filed https://github.com/bazelbuild/rules_nodejs/issues/593

**Pull request merged**